services/status/status.proto

```diff
@@ -0,0 +1,11 @@
+message Request{
+
```

Comment:
Seems like we have to manually define an extension so it's available to others. https://developers.google.com/protocol-buffers/docs/proto#extensions

Anyway, it's probably not important right now so it's ok.